docs/release-notes/1.5.3/release-1.5.3.md

```diff
@@ -0,0 +1,41 @@
+# [ML.NET](http://dot.net/ml) 1.5.3
+
+## **New Features**
+- **New API for exporting models to Onnx**. ([#todo](https://github.com/dotnet/machinelearning/pull/todo)). A new API has been added to Onnx converter to specify the output columns you care about. This will export a smaller and more performant model in many cases.
+
+## **Enhancements**
+- Perf improvement for TopK Accuracy and return all topK in Classification Evaluator ([#5395](https://github.com/dotnet/machinelearning/pull/5395))
+- Update OnnxRuntime to 1.6 ([#5529](https://github.com/dotnet/machinelearning/pull/5529))
+- Updated tensorflow.net to 0.20.0 ([#5404](https://github.com/dotnet/machinelearning/pull/5404))
+
+## **Bug Fixes**
+- **AutoML.NET specific fixes**.
+  - Fixed AutoFitMaxExperimentTimeTest ([#5506](https://github.com/dotnet/machinelearning/pull/5506))
+  - Fixed code generator tests failure ([#5520](https://github.com/dotnet/machinelearning/pull/5520))
+  - Use Timer and ctx.CancelExecution() to fix AutoML max-time experiment bug ([#5445](https://github.com/dotnet/machinelearning/pull/5445))
+  - Handled exception during GetNextPipeline for AutoML ([#5455](https://github.com/dotnet/machinelearning/pull/5455))
+  - Fixed issue when parsing float string fails on pl-PL culture set using Regression Experiment ([#5163](https://github.com/dotnet/machinelearning/pull/5163))
```

Comment:
Expanding the wording, as it fixed many cultures, and all experiment types:
```suggestion
  - Fixed internalization bug in AutoML parameter sweeping caused by culture dependent float parsing ([#5163](https://github.com/dotnet/machinelearning/pull/5163))
```

Reply:
Perhaps could also mention the bug, previously, was stopping AutoML from working in various cultures, like pl-PL and ar-SA.